wca-guidelines.md

```diff
@@ -63,7 +63,7 @@ To be more informative, each Guideline is classified using one of the following
 - 3a+++) [CLARIFICATION] Competitors may borrow puzzles from other competitors privately, to use in competition.
 - 3a1+) [CLARIFICATION] Competitors may be disqualified if they do not have a puzzle ready to submit soon after they are called (e.g. if they planned to use a puzzle that another competitor is currently using, and therefore cannot submit a puzzle at the moment).
 - 3d1b+) [CLARIFICATION] Patterns do not need to be present on every face, but only on faces where a distinct color cannot be found. Pieces must not have any features (e.g. textures, pattern irregularities) that significantly distinguish them from similar pieces.
-- 3h+) [CLARIFICATION] Puzzles may be refined internally by sanding or lubricating.
+- 3h+) [CLARIFICATION] Puzzles may be refined internally by sanding, lubricating or modifications which improve the stability (e.g. magnets).
```

Comment:
What's the convention on the oxford (serial) comma? It looks like `3h++` below does use it, but you did not use it here.

Reply:
We use the serial comma: https://github.com/thewca/wca-regulations/blob/draft/style-guide.md#specific-formatting-conventions

Reply:
I would also just say "stability" instead of "the stability".

("the stability of the puzzle" is also clear, but I don't think think it's any better than just "stability".)